subseas_flex_fcst/config-bd.yaml

```diff
@@ -23,3 +23,21 @@ targets: null
 
 # Viz' set up
 zoom: 7
+
+# Bangladesh
+shapes_adm:
+       - name: Divisions
+         color: black
+         sql: select gid as key, name_1 as label, ST_AsBinary(the_geom) as the_geom
```

Comment:
You don't seem to use the key anywhere, so you could drop it from the query. If you think we're likly to need it in the future, I have no objection to keeping it.

Reply:
How are we going to select polygons in the future, to do for instance weight-average of the gridboxes over the polygons? Is that when it might be used?

Reply:
yes